lib/player.js

```diff
@@ -5135,6 +5243,13 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
           }
         }
       }
+      if (this.lcevcDil_) {
+        this.lcevcDil_.hideCanvas();
```

Comment:
If you have show/hide canvas methods, could you use those above where you were poking into the styles of the canvas directly?

Reply:
Yes we have removed all direct references and redirected them through these functions.